packages/hyperverse-ethereum-token/source/useToken.ts

```diff
@@ -24,6 +24,9 @@ function TokenState(initialState: { tenantId: string } = { tenantId: TENANT_ADDR
 	useEffect(() => {
 		const fetchContract = async () => {
 			const proxyAddress = await contract.getProxy(tenantId);
+			if(proxyAddress == '0x0000000000000000000000000000000000000000') {
```

Comment:
Can we check if there is a variable for the 0x000 in eithers? Would look prettier than trying to count all the zeros